src/components/ContactCard/ContactForm/formValuesToContact.js

```diff
@@ -65,13 +65,14 @@ const formValuesToContact = (data, oldContact) => {
       }
     },
     metadata: {
-      ...get(oldContact, 'metadata', {}),
       version: 1,
       cozy: true
     }
   }
 
-  return updateIndexFullNameAndDisplayName(contactWithFormValues)
+  return updateIndexFullNameAndDisplayName(
+    merge({ ...oldContact }, contactWithFormValues)
```

Comment:
Oh, didn't know that lodash.merge was recursive! 👍

Reply:
yes, and it mutates... so that's why `{...oldContact}` and not simply `oldContact`